packages/schemas/tables/_before_all.sql

```diff
@@ -1,3 +1,3 @@
 /* This SQL will run before all other queries. */
 
-create role logto_tenant_${database} noinherit;
+create role logto_tenant_${database} password '${password}' noinherit;
```

Comment:
nit: `create role ${role_name} password '' noinherit;` could sometimes lead to PostgreSQL warning, better keep the query simple (e.g. `create role ${role_name} noinherit;`) when the `--encrypt-base-role` flag is false.